scripts/stubtest/allowlist_todo.txt

```diff
@@ -969,28 +935,13 @@ django.db.models.fields.json.KeyTransform.as_sqlite
 django.db.models.fields.json.KeyTransformIsNull.as_sqlite
 django.db.models.fields.json.KeyTransformNumericLookupMixin.process_rhs
 django.db.models.fields.related.ForeignKey.__class_getitem__
-django.db.models.fields.related.ForeignKey.contribute_to_related_class
 django.db.models.fields.related.ForeignKey.convert_empty_strings
 django.db.models.fields.related.ForeignKey.descriptor_class
 django.db.models.fields.related.ForeignKey.formfield
 django.db.models.fields.related.ForeignKey.get_db_converters
-django.db.models.fields.related.ForeignObject.__copy__
 django.db.models.fields.related.ForeignObject.__get__
-django.db.models.fields.related.ForeignObject.contribute_to_related_class
-django.db.models.fields.related.ForeignObject.forward_related_accessor_class
-django.db.models.fields.related.ForeignObject.get_class_lookups
-django.db.models.fields.related.ForeignObject.get_extra_descriptor_filter
-django.db.models.fields.related.ForeignObject.get_extra_restriction
-django.db.models.fields.related.ForeignObject.get_foreign_related_value
-django.db.models.fields.related.ForeignObject.get_instance_value_for_fields
-django.db.models.fields.related.ForeignObject.get_joining_columns
-django.db.models.fields.related.ForeignObject.get_local_related_value
-django.db.models.fields.related.ForeignObject.get_path_info
-django.db.models.fields.related.ForeignObject.get_reverse_joining_columns
-django.db.models.fields.related.ForeignObject.get_reverse_path_info
 django.db.models.fields.related.ForeignObject.path_infos
-django.db.models.fields.related.ForeignObject.related_accessor_class
-django.db.models.fields.related.ForeignObject.requires_unique_target
+django.db.models.fields.related.ForeignObject.get_extra_restriction
```

Comment:
2. Let's add other missing attributes to `ForeignObject`

Reply:
I have two questions:

1. I tried to define the `get_extra_restriction` method, but I encountered an error due to the parameter mismatch with the subclass. I attempted to use `type: ignore` on the method in the subclass, but it was not allowed. What steps can I take to resolve this issue?

2. Should I also add the cached_property method?